src/test/java/roomescape/controller/ReservationControllerTest.java

```diff
@@ -0,0 +1,86 @@
+package roomescape.controller;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.springframework.http.ResponseEntity;
+import roomescape.domain.Reservation;
+import roomescape.domain.ReservationTime;
+import roomescape.dto.ReservationResponse;
+import roomescape.repository.fake.FakeReservationRepository;
+import roomescape.repository.fake.FakeReservationTimeRepository;
+
+class ReservationControllerTest {
+    FakeReservationRepository fakeReservationRepository = new FakeReservationRepository();
+    FakeReservationTimeRepository fakeReservationTimeRepository = new FakeReservationTimeRepository();
+    private final ReservationController reservationController = new ReservationController(fakeReservationRepository, fakeReservationTimeRepository);
+
+    private void makeStubReservationTime(Long id, LocalTime startAt) {
+        fakeReservationTimeRepository.createReservationTime(new ReservationTime(id, startAt));
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = {"히스타, 2002-12-02, 1, 18:00"})
+    void create(String nickname, LocalDate date, Long timeId, LocalTime time) {
+        // given
+        makeStubReservationTime(timeId, time);
+        Reservation reservation = new Reservation(null, nickname, date, timeId);
+
+        // when
+        ResponseEntity<ReservationResponse> actualResponse = reservationController.create(reservation);
+
+        // then
+        Assertions.assertThat(actualResponse).isNotNull();
```

Comment:
```suggestion
        assertThat(actualResponse).isNotNull();
```

`Assertions.`가 반복되네요. stati import 처리하면 어떨까요?

Reply:
좋은 생각입니다 :)
근데 저는 `Assertions.` 을 반복해서 붙이는 게 귀찮고 어느정도의 노력이 필요하다고 느껴본적이 없어서 사실 굳이 `static import`를 해야할까? 싶긴 해요. 사실 매 테스트 클래스를 생성할때마다 `static import`를 붙이는 게 더 번거롭다고 생각했어요.

만약 컨벤션이라면 따르겠지만 아직까지 큰 불편함을 느끼지 못했고, `static import`보다 클래스명을 앞에 적어주는 방식이 명확성도 높다고 생각하기에 일단은 수정하지 않겠습니다.

Reply:
`static import`를 하면 IDE가 이를 인식해서 연관된 메서드를 보여주기도 해요. 생산성을 고려해보면 `static import`가 좋아보이는데, 어떻게 생각하시나요?


<img width="684" alt="image" src="https://github.com/user-attachments/assets/bbbe5cb0-f014-49b4-bc78-ba0581f6fc10" />


<img width="689" alt="image" src="https://github.com/user-attachments/assets/24ce301a-5549-4a9b-bc46-53744e36444b" />

Reply:
자동으로 연관된 메서드를 보여주는 기능은 `static import` 없이도 동작하지 않나요?
제가 생각했을 때 `static import`의 좋은 점은 모든 `assertJ`의 코드 앞에 `Assertions.` 을 안 붙여도 되는 게 좋은 점인 것 같아요! (즉 타자수가 적다)

그치만 저는 아직까지는 앞에 `Assertions.` 를 붙이는 게 시간이나 번거로움이 든다고 느낀 적이 없어요.
 한번 다음 미션 때 `static import`를 적용해서 써보고 차이를 느껴볼게요 :) ㅎㅎ

Reply:
오?? `static import` 없이도 동작하나요?? 안된다는 점을 보이기 위해 위에 스샷을 첨부하긴 했는데, 음 IntelliJ 버전 차이려나요

Reply:
`As` 까지 쳤을 때에는 `Assertions..` `Assert`등이 떴다가.
`Assertions.` 를 해야만 `Assertions`에 관련된 내용들이 뜨긴 해요~

Reply:
그렇다면 히스타는 매번 `Assertions.`까지 입력한 후 원하는 메서드를 선택하고 있었군요. 개인 취향이기 때문에 스레드는 이만 종료하겠지만, 그러한 비용까지도 절약하면 생산성이 오를 것 같네요~